src/utils/index.ts

```diff
@@ -0,0 +1,105 @@
+import { Contract } from '@ethersproject/contracts'
+import { getAddress } from '@ethersproject/address'
+import { AddressZero } from '@ethersproject/constants'
+import { JsonRpcSigner, Web3Provider } from '@ethersproject/providers'
+import { BigNumber } from '@ethersproject/bignumber'
+import { abi as IUniswapV2Router02ABI } from '@uniswap/v2-periphery/build/IUniswapV2Router02.json'
+import { ChainId, JSBI, Percent, Token, CurrencyAmount, Currency, ETHER } from '@pancakeswap/sdk'
+import { ROUTER_ADDRESS } from '../config/constants'
+import { BASE_BSC_SCAN_URLS } from '../config'
+import { TokenAddressMap } from '../state/lists/hooks'
+
+// returns the checksummed address if the address is valid, otherwise returns false
```

Comment:
If this file `src/utils/index.ts` is used entirely by exchange code, perhaps its name should reflect that?